VSocks.md

```diff
@@ -0,0 +1,133 @@
+# Kata Containers and vsocks
+
+- [Introduction](#introduction)
+    - [proxy communication diagram](#proxy-communication-diagram)
+    - [vsock communication diagram](#vsock-communication-diagram)
+- [System requirements](#system-requirements)
+- [Advantages of using vsocks](#advantages-of-using-vsocks)
+    - [High density](#high-density)
+    - [Reliability](#reliability)
+
+## Introduction
+
+There are two different ways processes in the virtual machine can communicate
+with processes in the host. The first one is by using serial ports, where the
+processes in the virtual machine can read/write data from/to a serial port device
+and the processes in the host can read/write data from/to a unix socket. This
+solution has been part of QEMU from long time ago and almost all GNU/Linux
```

Comment:
Lines 16 and 17  - Remove "This solution ... long time ago and almost all" and begin sentence with "Most GNU/Linux ..."